workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/commerce-catalogs/catalog.products.specifications.json

```diff
@@ -1,11 +1,11 @@
 [
 	{
 		"categoryKey": "product-metadata",
-		"cpDefinitionExternalReferenceCode": "MKP-APP-SETTINGS",
-		"description": "Define your app settings.",
-		"facetable": false,
-		"key": "app-settings",
-		"title": "App Settings"
+		"cpDefinitionExternalReferenceCode": "MKT-TECHNOLOGY-INTEGRATION",
+		"description": "Technology Integration",
+		"facetable": true,
+		"key": "technology-integration",
```

Comment:
@welitonpe can you please rename this?

Sorry for the extra work

Use this as reference PARTNERSHIP_TYPE

partnership-type

Partnership Type

This will be used also in the Account Custom Field that Caleb is working